queenbee/config/auth.py

```diff
@@ -0,0 +1,94 @@
+import json
+from enum import Enum
+from urllib import request
+from urllib.error import HTTPError
+from typing import Union
+from pydantic import Field, SecretStr
+
+
+from ..base.basemodel import BaseModel
+from ..base.request import make_request, USER_AGENT_STRING
+
+
+class BaseAuth(BaseModel):
+
+    domain: str = Field(
+        ...,
+        description='The host domain to authenticate to',
+        example='api.pollination.cloud'
+    )
+
+    access_token: SecretStr = Field(
+        None,
+        description='A JWT token retrieved from a previous login'
+    )
+
+    @property
+    def auth_header(self):
+        if self.access_token is None:
+            return ''
+        return f'Bearer {self.access_token.get_secret_value()}'
+
+    def refresh_token(self):
+        pass
+
```

Comment:
2 lines!

Reply:
?

Reply:
You need two empty lines before the class. There was only one. PEP8 🚨